### ISSUE TITLE
"Disable SQL code" can generate bad syntax

### ISSUE THREAD

**Issue by icunnyngham** [pgmodeler/pgmodeler#276]
First of all, thanks for this very nice tool!  There are a ton of bugs, and crashes which I could report, but this one seems very basic, so I'll start here:

When you "Disable SQL code", this is achieved by commenting out the relevant lines.  The problem with this method comes when the lines you are disabling are the last arguments in a "line".  For instance:

CREATE TABLE foo.bar(
        "ID" integer,
        CONSTRAINT "ID_pk" PRIMARY KEY ("ID"),
--  CONSTRAINT foo_to_qux FOREIGN KEY ("ID")
);

TL;DR: A spurious comma is introduced into the SQL when disabled code happens to be at the end of a block.
